packages/table/src/Cell/Cell.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { PropsWithChildren } from 'react';
+import InternalCellWithoutVS from './InternalCellWithoutVS';
+import InternalCellWithVS from './InternalCellWithVS';
+import { CellProps } from './types';
+
+const Cell = <T extends unknown>({
+  cell,
+  toggleExpandedIconProps,
```

Comment:
Fails when not provided. I tried breaking it out into `Cell`, `InternalCellWithVS`, `InternalCellWithoutVS`, and `InternalCellBase` and creating separate types for them, but it broke. Can't remember what broke then. Will need to come back to this.